pkg/utils/gpg_test.go

```
@@ -0,0 +1,111 @@
+
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/ProtonMail/go-crypto/openpgp/armor"
+)
+
+// testDataSignedChecksum is a real sha256sum.txt.gpg fetched from
+// mirror.openshift.com's oc client release directory. It is a real,
+// Red Hat-signed artifact used to validate that VerifyAndExtractSignedMessage
+// works against production data, not a synthetic/mocked signature.
+const testDataSignedChecksum = "testdata/oc-sha256sum.txt.gpg"
+
+func TestVerifyAndExtractSignedMessage_ValidSignature(t *testing.T) {
```

golangci-lint (2.12.2)

[error] Function TestVerifyAndExtractSignedMessage_ValidSignature missing the call to method parallel


(paralleltest)